### ISSUE TITLE
Does the warp-ctc implement a softmax layer?

### ISSUE THREAD

**Issue by codyhan** [baidu-research/warp-ctc#86]

**Comment by freesouls** [comment]
yes, inside the warp-ctc these is softmax, for example in `cpu_ctc.h` you can see  [cpu softmax](https://github.com/baidu-research/warp-ctc/blob/master/include/detail/cpu_ctc.h#L79)
